src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/resource/publicApi/model/licence/Licence.kt

```diff
@@ -77,4 +77,7 @@ data class Licence(
 
   @Schema(description = "The AP and PSS conditions that form the licence")
   val conditions: Conditions,
+
+  @Schema(description = "Is the licence in the hard stop period? (Within two working days of release)")
+  val isInHardStopPeriod: Boolean,
```

Comment:
probably don't want it on the public api? bit of an internal impl detail

Reply:
That's fair, I thought it may be helpful for external users to know when a CA is able to create the licence, but I suppose the main thing would be if the CA actually does, which we can expose in other ways